### DISCUSSION TITLE
Disable ligatures on cursor

### DISCUSSION THREAD

**Discussion by idr4n** [wezterm/wezterm#2607] [Q&A] [answered]
Is it possible to disable ligatures only when the cursor is on top of that particular character. I do like ligatures but sometimes it is difficult to edit them when you cannot see the real characters. Kitty has the option `disable_ligatures cursor` for this.

Otherwise, I'm just disabling ligatures all together with `	harfbuzz_features = { "calt=0", "clig=0", "liga=0" }` using Fira Code.

**Comment by wez** [comment, marked as answer]
See: https://github.com/wez/wezterm/issues/2181#issuecomment-1166398339

**Reply by idr4n** [reply]
Great, thanks!
